### PR TITLE
770 issue templates

### PR DESCRIPTION
**Pull request by bossenti**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+  - type: dropdown
+    attributes:
+      label: Affected StreamPipes components
+      description: >
+        Which StreamPipes components are affected by this bug?
+        Where did the bug occur?
+      multiple: true
+      options:
+        - Backend
```
Comment (dominikriemer) [inline .github/ISSUE_TEMPLATE/bug_report.yml:58]: Somehow the mapping from these components to "core/extensions" went wrong during issue migration.
Should we use these labels anyway?
E.g., "Core", "Extensions (pipeline elements and adapters)"?

Comment (bossenti) [reply .github/ISSUE_TEMPLATE/bug_report.yml:58]: I have already cleaned them up and corrected the "wrong" assignments.
So we can use the labels as planned.
Are you missing any other options here?
Or in general another input field?

**Review by tenthe — APPROVED** [review]
@bossenti awsome work. I really like the structure.
I think we will find out what we are missing when we use it.

**Review by RobertIndie — APPROVED** [review]
Awesome! LGTM

**Pull request merged**